WalletWasabi/Services/CcjClient.cs

```diff
@@ -505,7 +505,9 @@ private async Task TryRegisterCoinsAsync(CcjClientRound inputRegistrableRound)
 						var coin = State.GetSingleOrDefaultFromWaitingList(coinReference);
 						if (coin == null) throw new NotSupportedException("This is impossible.");
 						roundRegistered.CoinsRegistered.Add(coin);
+						coin.CoinJoinInProgress = false;
 						State.RemoveCoinFromWaitingList(coin);
+						CoinDequeued?.Invoke(this, coin);
```

Comment:
These are no good. If they are removed from the waiting list, it doesn't mean they are dequeued. They are just removed from the waiting list. From the waiting list, they can go to either being dequeud, or to a being mixed. I fix it. Also I'm not sure what does it would've made faster.

Reply:
This is part of the same mistake. I pushed it by mistake.